### ISSUE TITLE
Installed packages file can break backup jobs

### ISSUE THREAD

**Issue by ccztux** [ccztux/glsysbackup#59]
If installed_packages_filename is configured multiple times in various config files and those jobs were started nearly at the same time, this will cause glsysbackup to break.

**Comment by ccztux** [comment]
Maybe we should move the configuration variable: installed_packages_filename from config file to main script and generate a filename like this:
```
${script_name}.${script_config_name}.installed_packages.txt
```

**Comment by ccztux** [comment]
Fixed with: 015115f74b1610a47d42c5d7ffbd37950bd570ee
